ui/cryptomaterial/dropdown.go

```diff
@@ -284,8 +285,6 @@ func (d *DropDown) closedLayout(gtx C, iLeft int, iRight int) D {
 				layout.Rigid(func(gtx C) D {
 					return d.layoutOption(gtx, d.selectedIndex)
 				}))
-			w := (lay.Size.X * 800) / gtx.Dp(MaxWidth)
-			d.Width = w + 10
```

Comment:
I think removing this code caused this issue mentioned by @dreacot: https://github.com/crypto-power/cryptopower/pull/147#issuecomment-1747362071

I think this will fix the dropdown issue here. Consider replacing the deleted code with the suggested code below:  

```
if d.Width <= 0 {
    width := gtx.Dp(values.MarginPadding180)
    if d.revs {
	width = gtx.Dp(values.MarginPadding140)
    }
   d.Width = width
}
			
```

Also, this block of code is duplicated in `d.layoutOption(....)`. We can extract it into a `defaultDropdownWidth` method that returns the width based on the value of `d.revs`.

Reply:
This fix has been moved to #188